CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Stylable Components
+
+## Branch naming
+We use scheme in branch naming.
+It should contain branch author user name at first and branch name divided by `/`, and optional number of the issue. All words in branch name are separated by underscore: `_`
+`username/own_theme_park/42`
```

Comment:
I myself prefer `username/issue_number_if_relevant/branch_name`... easier for me to find branches that way (if I know who made it and the issue number, I don't need to know how he named the branch)